public/locales/en/translations.json

```diff
@@ -16,6 +16,7 @@
   },
   "errorFormat": "Error: {{message}}",
   "errorCode": {
+    "maxJobCountExceeded": "We were not able to create the job because the maximum job count has been exceeded.",
```

Comment:
I don't think we should have this in our translations since doing so opens the door to a world of problems. We now take the possible responsibility for storing translations strings for not just our own but error strings/status codes used by  other teams. Of course we then take a dependency on their error code.

Reply:
I hear you.  If we don't translate ourselves, then ( 1 ) messages will contain ugly/scary looking information, and ( 2 ) messages from dotNet and java are different. 
Also, do we have a way to tell the server what language to give us messages in?  Because if we don't, then there's a high probability of language mismatches.

Reply:
Here's what the dotnet message looks like now... without adding our own translation....  ![image](https://user-images.githubusercontent.com/16829221/39787035-2b519a42-52d8-11e8-85db-d4fd9def7d33.png)